config.yaml

```diff
@@ -21,6 +21,7 @@ default:
    --plugin-option=tmpfs:required_ram_mb=39800"
  commit_updates: True
  push_updates: True
+ build_dependencies: True
```

Comment:
I don't think using the commit ID as a primary key is a good idea. There are many other factors that may require rebuilding a package. Altering the spec file or modifying the build scripts, for example. I agree with checking if the dependencies are already built by checking if a directory `<package_name>` exists, but I think the default should be to always rebuild the dependencies and the decision not to do this should be mostly manual.